<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialLevelParam</class>
 <widget class="QWidget" name="SerialLevelParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_top" stretch="2,4,2,3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_select_players">
     <property name="title">
      <string>选择账号</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_select_player">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_select_1p">
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>房主：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_select_1p">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2P</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_select_2p">
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>房客：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_select_2p">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>无</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2P</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_file_path">
     <property name="title">
      <string>文件路径</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_file_path">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_series_path">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>序列文件：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_series_path">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_series_path">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_plan_path_team">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>组队方案：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineEdit_plan_path_team">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pushButton_plan_path_team">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_plan_path_1p">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>房主方案：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_plan_path_1p">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_plan_path_1p">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_plan_path_2p">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>房客方案：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="lineEdit_plan_path_2p">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pushButton_plan_path_2p">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_other_setting">
     <property name="title">
      <string>通关设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_other_setting" stretch="2,3,2,3">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_flop_pos">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>翻牌位置：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_flop_pos">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>178</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>1;2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_quest_panel">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>任务界面：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_quest_panel">
        <property name="minimumSize">
         <size>
          <width>178</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>无</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>美食大赛</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>探险营地</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_top">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
